tests/foreman/ui/test_host.py

```diff
@@ -136,6 +136,20 @@ def tracer_install_host(rex_contenthost, target_sat):
     return rex_contenthost
 
 
+@pytest.fixture
+def new_host_ui(target_sat):
```

Comment:
Maybe we could leverage the `setting_update` fixture instead. Somewhere I've even seen we can provide the value in the parametrize decorator and don't need to care in the test body, which would act the same like this fixture.
See https://github.com/SatelliteQE/robottelo/blob/285d7a6cce9890e65fb6c90f5602e5d810170d15/tests/foreman/api/test_settings.py#L42

Reply:
Ooh, I like this. Does it change the setting back afterwards though? that's important. I'll look at this.

Reply:
Yes it does! See https://github.com/SatelliteQE/robottelo/blob/83ca28419c129aad304d49c89f758af36b9a74b7/pytest_fixtures/component/settings.py#L20